aea/cli/registry/fetch.py

```diff
@@ -44,7 +44,7 @@
 
 
 @clean_after
-def fetch_agent(
+def fetch_agent_http(
```

Comment:
Please revert the name change, this might cause some breaking changes on the open-autonomy

Reply:
done